### PR TITLE
INFRA: Initialize ASP.NET Core Web API Project

### PR DESCRIPTION
**Pull request by MahalakshmiMurari**
I did the below changes. Please review and approve the request
1)Initialized core Web API project,
2)Added copyright header, gitignore 
3)Did code cleanup,
4)Followed standard naming convention,
5)Removed unnecessary comments

### CODE REVIEW

File: ReserveSpot.Services.Api/Controllers/WeatherForecastController.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+﻿// ------------------------------------------------
```
Comment (ShriHumrudha) [inline ReserveSpot.Services.Api/Controllers/WeatherForecastController.cs]: Remove the Controller file

File: ReserveSpot.Services.Api/ReserveSpot.Services.Api.csproj [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```
Comment (ShriHumrudha) [inline ReserveSpot.Services.Api/ReserveSpot.Services.Api.csproj]: Change it to 6.0. Add the configuration for disabling implicit namespace.

File: ReserveSpot.Services.Api/Program.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace ReserveSpot.Services.Api
+{
+    public class Program
+    {
+        public static void Main(string[] args) =>
+                 CreateHostBuilder(args).Build().Run();
```
Comment (ShriHumrudha) [inline ReserveSpot.Services.Api/Program.cs]: Fix the tab indendation

File: ReserveSpot.Services.Api/Program.cs [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+        public static IHostBuilder CreateHostBuilder(string[] args)
+        {
+            return Host.CreateDefaultBuilder(args)
+                .ConfigureWebHostDefaults(webBuilder =>
+                         webBuilder.UseStartup<Startup>());
```
Comment (ShriHumrudha) [inline ReserveSpot.Services.Api/Program.cs]: Fix the tab indendation

File: ReserveSpot.Services.Api/Startup.cs [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+
+namespace ReserveSpot.Services.Api
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration) =>
+                    Configuration = configuration;
```
Comment (ShriHumrudha) [inline ReserveSpot.Services.Api/Startup.cs]: Tab indendation

File: ReserveSpot.Services.Api/Startup.cs [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+        public IConfiguration Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddControllers();
+            services.AddSwaggerGen(options =>
```
Comment (ShriHumrudha) [inline ReserveSpot.Services.Api/Startup.cs:25]: Single & Multi-line code to be separated with new line.

File: ReserveSpot.Services.Api/Startup.cs [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+            });
+        }
+
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+                app.UseSwagger();
+                app.UseSwaggerUI(options =>
```
Comment (ShriHumrudha) [inline ReserveSpot.Services.Api/Startup.cs:46]: Single & Multi-line code to be separated with new line.

File: ReserveSpot.Services.Api/WeatherForecast.cs [hunk 8]
```diff
@@ -0,0 +1,19 @@
+// ------------------------------------------------
```
Comment (ShriHumrudha) [inline ReserveSpot.Services.Api/WeatherForecast.cs]: Remove this file.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Comment by MahalakshmiMurari** [comment]
CODE RUB:  Updated .NET version, Added the configuration for disabling implicit namespace. removed unnecessary files and fixed tab indentations.

**Comment by MahalakshmiMurari** [comment]
Please review and approve the pull request.

File: ReserveSpot.Services.Api/Program.cs [hunk 9]
```diff
@@ -0,0 +1,22 @@
+// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+using Microsoft.AspNetCore.Hosting;
```
Comment (ShriHumrudha) [inline ReserveSpot.Services.Api/Program.cs:5]: Please run code clean up before check-in.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: ReserveSpot.Services.Api/Program.cs (hunk 3 above)
Comment (MahalakshmiMurari) [reply ReserveSpot.Services.Api/Program.cs]: Fixed the tab indendation

File: ReserveSpot.Services.Api/WeatherForecast.cs (hunk 8 above)
Comment (MahalakshmiMurari) [reply ReserveSpot.Services.Api/WeatherForecast.cs]: Removed the above mentioned file

File: ReserveSpot.Services.Api/Startup.cs (hunk 7 above)
Comment (MahalakshmiMurari) [reply ReserveSpot.Services.Api/Startup.cs:46]: Seperated single and Multi-line code with new line

File: ReserveSpot.Services.Api/Program.cs (hunk 9 above)
Comment (MahalakshmiMurari) [reply ReserveSpot.Services.Api/Program.cs:5]: Ran Code Cleanup and also verified that build is successful without any errors and warnings

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**